cypress/e2e/signIn.cy.js

```diff
@@ -10,29 +10,37 @@ const homePage = new HomePageObject();
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  beforeEach(() => {
     cy.task('db:clear');
```

Comment:
The database is cleared before each test but there is no sign of its refill. It's recommended to refill the database with necessary data after clearing it.